content/docs/user-guide/contributing/docs.md

```diff
@@ -63,7 +63,9 @@ formatted and linted as well, which is also ensured by the full setup below.
 > dependencies, run `yarn build`, and start the webserver so that you can start
 > straight away:
 
-[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/iterative/dvc.org)
+<a href="https://gitpod.io/#https://github.com/iterative/dvc.org">
+  <img src="https://gitpod.io/button/open-in-gitpod.svg" alt="Open in Gitpod">
+</a>
```

Comment:
What if we eant to put the button inside the block quote above? It's an actual formatting change in my regular updates PR. Will it work?

Reply:
I.e. something like
```suggestion
> straight away:
>
> <a href="https://gitpod.io/#https://github.com/iterative/dvc.org">
>   <img src="https://gitpod.io/button/open-in-gitpod.svg" alt="Open in Gitpod">
> </a>
```

Reply:
Surprisingly, this worked with no issue!

Reply:
I see that. Noice